### PR TITLE
Migrate from Cloud Foundry to Docker/Kube

### PR DESCRIPTION
**Pull request by ddgenome**
Remove Cloud Foundry trappings and add those for Docker.  Update
Travis CI build configuration and script.  Add Kube deployment spec.
Update README.

Add AutomationGenerator.  Remove NodeGenerator.  Use owner and repo
name for package name.

Add HelloAutomation command handler.

Towards #3 #5 #11 #12

### CODE REVIEW

**Review by jessitron — APPROVED** [review]
this looks beautiful thank you

File: scripts/travis-build.bash [hunk 1]
```diff
@@ -281,6 +315,22 @@ function cf-push () {
     fi
 }
 
+# sync build to S3 bucket
+# usage: s3-sync BUCKET
```
[the remark this answers is not in the document]
Comment (ddgenome) [reply scripts/travis-build.bash]: Publishes a "site" to S3. It does nothing in this repository but this is just our standard NPM Travis build script.

**Review by cdupuis — APPROVED** [review]
LGTM

**Comment by jessitron** [comment]
Pull request auto merged by Atomist.

* 2 approved reviews by @cdupuis, @jessitron
* 5 successful checks

[atomist:generated] [atomist:auto-merge]

**Pull request merged**